### PR TITLE
added few functions and update main

### PR DESCRIPTION
**Pull request by lytvynov-kostiantyn**
* added "get_float" func
* added "get_int" func
* update main (added possibility of finding a factorial)

### CODE REVIEW

File: calcul.py [hunk 1]
```diff
@@ -1,5 +1,24 @@
-import math
-from math import *
+from math import sqrt, factorial
```
Comment (lytvynov-kostiantyn) [inline calcul.py:1]: I think it makes no sense to download the entire library for the sake of two functions

File: calcul.py [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+
+
+# Getting float from user
+def get_float(phrase='Введення: '):
```
Comment (lytvynov-kostiantyn) [inline calcul.py:5]: I think these functions will be useful to our program

File: calcul.py [hunk 3]
```diff
@@ -33,38 +52,42 @@ def calc_korin_kyb(a):
...
             print(calc_stypin(a, b))
         elif choice == 6:
-            a = int(input('Введіть число: '))
+            a = get_float('Введіть число: ')
             print(calc_sqrt(a))
         elif choice == 7:
-            a = int(input('Введіть число: '))
+            a = get_float('Введіть число: ')
             print(calc_korin_kyb(a))
         elif choice == 8:
```
Comment (lytvynov-kostiantyn) [inline calcul.py:83]: I propose to expand the functionality by adding the ability to find factorial

File: calcul.py (hunk 1 above)
Comment (Pavel8919) [reply calcul.py:1]: awersome! well done!

**Pull request merged**